addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -524,8 +524,8 @@ QUnit.test('chat window: basic rendering', async function (assert) {
     );
     assert.strictEqual(
         chatWindowHeader.querySelectorAll(`:scope .o_ChatWindowHeader_command`).length,
-        2,
-        "should have 2 commands in header part"
+        3,
+        "should have 3 commands in header part"
```

Comment:
could be worth adding assertion for the additional command,
I know long-term we will have 0 tests, but considering current tests act as specs, here it's not clear why it's 3 when only 2 exact named commands are asserted below (would expect the 3rd one).

This or we remove the awareness of amount of commands in this test. Up to you.

Reply:
I'll add the assertion.